pyproject.toml

```diff
@@ -13,8 +13,8 @@ additional-compiled-static-assets = [
 ]
 
 [project]
-name = "pydata-sphinx-theme"
-description = "Bootstrap-based Sphinx theme from the PyData community"
+name = "neuroinformatics-sphinx-theme"
+description = "Bootstrap-based Sphinx theme from the neuroinformatics community"
```

Comment:
I'd change this to ""Bootstrap-based Sphinx theme for the Neuroinformatics Unit, based on the PyData theme."